deepchecks/base/display_suite.py

```diff
@@ -70,17 +91,34 @@ def display_suite_result(suite_name: str, results: List[Union[CheckResult, Check
         unique_id = ''
     else:
         unique_id = get_random_string()
+    is_widgets = is_widgets_enabled()
+    if is_widgets:
```

Comment:
IMO having many `if is_widgets` makes the code very hard to read. 
I'd prefer to have 2 functions, display_as_widget, display_as_html even in the price of some code duplication

Reply:
It means any update to the code will be very hard

Reply:
I disagree, right now every update is very hard since the function is very complicated

Reply:
I remove some if is_widgets

Reply:
there aren't many differences in the logic, I don't think we should want to split it

Reply:
@ItayGabbay do you have opinion?

Reply:
I think we should separate it to functions

Reply:
done